scripts/build-manifests.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
```

Comment:
I think this can be done now, so we do not have to change it until we create the Python CLI.